### DISCUSSION TITLE
Does index.umd.production.min.js includes IE11 compatibility for CDN Usage

### DISCUSSION THREAD

**Discussion by ajaybeniwal** [react-hook-form/react-hook-form#2064] [Q&A] [answered]
We are having a micro front end setup and would like to share the react-hooks-form across all projects so the question is whether umd build is Ie11 compatible and is is okay to include modern build instead of IE11 build if we are loading entire corejs at global scope on application load

**Comment by bluebill1049** [comment, marked as answer]
no it doesn't :(

**Reply by ajaybeniwal** [reply]
Is it possible to publish the UMD compatible build with IE11 supports because its very critical for us to share the library with IE11 support across Microfrontends

**Reply by ajaybeniwal** [reply]
Sorry for unmarking the answer is it possible to publish the module

**Reply by bluebill1049** [reply]
sorry @ajaybeniwal, don't think we will bundle UMD version for IE compatibility.
